app/assets/src/helpers/taxon.js

```diff
@@ -22,3 +22,19 @@ export const getGeneraContainingTags = taxInfoArray => {
   }
   return generaContainingTags;
 };
+
+export const getGeneraPathogenCounts = speciesCounts => {
+  const generaPathogenCounts = {};
+  for (let speciesInfo of Object.values(speciesCounts)) {
```

Comment:
Style note: We have been using `lodash/fp` like :`values(speciesCounts).forEach(`.
Up to you if you want to change.